docs/backends.js

```diff
@@ -183,7 +188,7 @@ var circleBackend = function(settings, resultCallback) {
             }
           })
         )
-      }, [])
+      }, []).filter(function (build) { return !!build })
```

Comment:
Removes undefined array entries, caused by the added escape clause

Reply:
Otherwise this PR is good but make filtering BEFORE map. Like this
`Object.keys(repository.branches).filter(/* put filter logic here */).map(/* same logic here */)`